api/modules/projects/controller.js

```diff
@@ -11,28 +11,72 @@ var PublishedFiles = require('../publishedFiles/model');
 var Model = require('./model');
 var controller = new BaseController(Model);
 
+// Taken from http://stackoverflow.com/a/643827
+function isDate(val) {
+  return Object.prototype.toString.call(val) === '[object Date]';
+}
+
+function formatResponse(model) {
+  var created = model.get('date_created');
+  var updated = model.get('date_updated');
+
+  if (isDate(created)) {
+    model.set('date_created', created.toISOString());
+  }
+  if (isDate(updated)) {
+    model.set('date_updated', updated.toISOString());
+  }
+
+  return model;
+}
+
 controller.formatRequestData = function(req) {
+  var now = new Date();
```

Comment:
not in this case. This method is called before any server-side processing happens. And I think you would want to work with date objects when you process the request. I'll change the database level interaction that happens so that just before it communicates with the db, it converts the dates to ISO strings